geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/docker-compose-lib.yaml

```diff
@@ -285,7 +285,7 @@ services:
     image: camptocamp/geomapfish-tools:${GEOMAPFISH_VERSION}
     restart: unless-stopped
     volumes:
-      - ./geoportal/${PACKAGE}_geoportal/locale:/app/${PACKAGE}_geoportal/locale
+      - ./geoportal:/app/geoportal
```

Comment:
This seems wreid to me.
In tools container, I'm use to have installed "package"_geoportal package in /app (useful for debugging as we can substitute geoportal by tools and we have more dev tools).
And all sources in /src/ for development tasks (generate alembic revisions, update po files, etc).